app/models/job_invocation.rb

```diff
@@ -42,4 +45,37 @@ def failed_hosts
   def locks_for_resource(tasks, resource_type)
     tasks.map { |task| task.locks.where(:resource_type => resource_type).first }.compact
   end
+
+  def total_hosts_count
+    if targeting.resolved_at.nil?
+      _('N/A')
+    else
+      targeting.hosts.count
+    end
+  end
+
+  def execution_type
```

Comment:
this wasn't resolved

@inecas:
> I know I've came initially with this naming, but I'm still not sure that's the best name for the value we are collecting, Anyone suggestions on better name?

`%w(execution_mode invocation_time timing trigger_type trigger_mode).sample`